### DISCUSSION TITLE
请问上传文件后报错：Local and remote file hash DOESN'T match是什么原因?

### DISCUSSION THREAD

**Discussion by lijiatin** [houtianze/bypy#635] [Q&A] [answered]
提示是
<I> [15:05:33] Local and remote file hash DOESN'T match
<E> [15:05:33] 'F:/tick_data/20230110/20230110.tgz' =C=> '/apps/bypy/20230110.tgz' FAILED.

是上传过程中网络原因的数据缺失吗？解压查看一个文件发现能打开，但是文件比较多，没法一一测试，也不知道数据是不是完整...期盼解答，感谢。

**Comment by houtianze** [comment, marked as answer]
这个错误一般可以忽略，原因是百度那边md5验证有时候有问题，有时候再传一次就可以了。可以加 --verify=False 略过md5比较。bypy compare可以比较本地和远程文件夹是否一致。

**Comment by death4444** [comment]
![image](https://github.com/user-attachments/assets/87e93682-03a4-4b6f-9339-46327f829ef5)
视频文件始终无法跳过hash验证

**Reply by houtianze** [reply]
> ![image](https://private-user-images.githubusercontent.com/26320543/415280179-87e93682-03a4-4b6f-9339-46327f829ef5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDM0Mjc2MDgsIm5iZiI6MTc0MzQyNzMwOCwicGF0aCI6Ii8yNjMyMDU0My80MTUyODAxNzktODdlOTM2ODItMDNhNC00YjZmLTkzMzktNDYzMjdmODI5ZWY1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAzMzElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMzMxVDEzMjE0OFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTk5MGUwM2YyNmQwOTg5OGMzZjRiODcxZTdmMmMwOTc3MzM1NTZjMWFhNTNhZjdlODlmZmIzODNkNzZiOWVmMTQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.5hKSvD57a8aeipKWUvpjcWpbxwtO2WD5OgufBL660jo) 视频文件始终无法跳过hash验证

可以看一下盘上边是不是已经上传成功？成功就可以忽略这个，这个hash不对是个老问题，一直没找到什么比较好的解决办法。
